dnSpy/dnSpy.Decompiler/MSBuild/ProjectCreatorOptions.cs

```diff
@@ -78,9 +78,9 @@ sealed class ProjectCreatorOptions {
 		public Func<TextWriter, IDecompilerOutput> CreateDecompilerOutput { get; set; }
 
 		/// <summary>
-		/// Set to 'true' to generate the more modern SDK style project files.
+		/// Set to 'true' to generate SDK style project files.
 		/// </summary>
-		public bool GenerateSDKStyleProjects { get; set; }
+		public bool SdkProject { get; set; }
```

Comment:
Please avoid changing the public API. Keep this as `GenerateSDKStyleProjects` but the console parameter as `--sdk-project`